Source/Urho3D/RmlUI/RmlCanvasComponent.cpp

```diff
@@ -101,7 +101,7 @@ void RmlCanvasComponent::SetUISize(IntVector2 size)
     {
         RenderSurface* surface = texture_->GetRenderSurface();
         surface->SetUpdateMode(SURFACE_MANUALUPDATE);
-        offScreenUI_->SetRenderTarget(surface, Color::BLACK);
+        offScreenUI_->SetRenderTarget(surface, Color::TRANSPARENT_BLACK);
```

Comment:
Can it be an attribute? I am okay with having transparent as default.

Reply:
Probably. TRANSPARENT_BLACK is used elsewhere though, in ClearTexture at line 149. I kinda thought SetUISize using Black was just an oversight.

Reply:
This probably is an oversight, but if we change the behavior and the flexibility is almost free, I would rather have it than not.

Reply:
I added a Clear Color attribute with a default value of TRANSPARENT_BLACK, and fixed an apparent bug in SetTexture. How does that look?